task-launcher/src/tasks/shared/helpers/initConfig.js

```diff
@@ -34,30 +35,33 @@ export const initSharedConfig = async (firekit, gameParams, userParams, displayE
     keyHelpers,
     age,
     maxTime, // maximum app duration in minutes
+    roarDefaults,  
     // storyCorpus,
     // story,
   } = cleanParams;
 
   const config = {
     userMetadata: { ...userMetadata, age },
     audioFeedback: audioFeedback || 'neutral',
-    skipInstructions: skipInstructions ?? true,
+    skipInstructions: skipInstructions,
     startTime: new Date(),
     firekit,
     displayElement: displayElement || null,
-    sequentialPractice: sequentialPractice ?? true,
-    sequentialStimulus: sequentialStimulus ?? true,
+    sequentialPractice: sequentialPractice,
+    sequentialStimulus: sequentialStimulus,
     // name of the csv files in the storage bucket
     corpus: corpus,
+    storeItemId: storeItemId,
     buttonLayout: buttonLayout || 'default',
-    numberOfTrials: numberOfTrials ?? 20,
-    task: taskName ?? 'egma-math',
+    numberOfTrials: numberOfTrials ?? roarDefaults ? null : 20,
+    task: taskName ?? roarDefaults ? 'trog' : 'egma-math',
```

Comment:
One of these changes is breaking the tasks corpus fetching and parsing. Can you please leave our defaults and pass ROAR configurations in?
```suggestion
    numberOfTrials: numberOfTrials ??  20,
    task: taskName ??  'egma-math',
```

Reply:
@tonyamurray , for example with this change just running the default task (`http://localhost:8080/`), which should be `egma-math`, attempts to load the TROG corpus, resulting in errors.